src/python/aqueduct_executor/operators/connectors/data/snowflake.py

```diff
@@ -1,11 +1,50 @@
+from typing import Any, Callable, Dict, List, Optional
+
 from aqueduct_executor.operators.connectors.data import config, relational, utils
+from aqueduct_executor.operators.connectors.data import connector, extract, load
+from aqueduct_executor.operators.utils.enums import ArtifactType
 from sqlalchemy import create_engine, engine
+from pyspark.sql import SparkSession, DataFrame
```

Comment:
Is this backwards compatible with current snowflake integrations in use? Or will it break if `pyspark` is not installed?